features/index_page.feature

```diff
@@ -0,0 +1,12 @@
+Feature: index page with a list restaurants
+  As a visitor
+  As soon as I arrive at the index page
+  I would like to be presented with some examples of restaurants that deliver food
+
+  Scenario: viewing list of restaurants
+    Given I am on the index page
+    And I should see "List Of Restaurants"
+    And I should see "Thai Restaurant"
+    And I should see "Italian Restaurant"
+    And I should see "Chinese Restaurant"
+    And I should see "Indian Restaurant"
```

Comment:
I would refactor this to something that you are actually going to display on your index page. Like "Welcome to Slow Food!" or something like that. You are going to replace "list of restaurants" with an actual list of restaurants, so why test for it now?